CHANGELOG.md

```diff
@@ -5,6 +5,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.37.0](https://github.com/buildkite/agent/compare/v3.36.1...v3.37.0) (2022-07-06)
```

Comment:
Aside: We seem to link to the compare or the tree for different versions. Perhaps in a future PR we should make them all the same.

Reply:
Ah yeah thanks, I've updated it to match the recent examples.

We use this fork of `ghch`, and it looks like I didn't use the fork this time?
https://github.com/Songmu/ghch/compare/main...buildkite:ghch:master

The whole release workflow needs some love. I'd love us to use https://github.com/release-drafter/release-drafter instead of `ghch` but I haven't investigated how well that'd work for us.